docs/user/metal/customization_ipi.md

```diff
@@ -12,7 +12,8 @@
 `externalBridge` | `baremetal` | The name of the bridge of the hypervisor attached to the external network. |
 `provisioningBridge` | `provisioning` | The name of the bridge on the hypervisor attached to the provisioning network. |
 `provisioningNetworkCIDR` | `172.22.0.0/24` | The CIDR for the network to use for provisioning. |
-`provisioningDHCPExternal` | `false` | Flag indicating that DHCP for the provisioning network is managed outside of the cluster by existing infrastructure services. |
+`provisioningNetwork` | `Managed` | ProvisioningNetwork is used to indicate if we will have a provisioning network, and how it will be managed. May be "Managed" (default), "Unmanaged," or "Disabled."
```

Comment:
The verb tense here doesn't match the other descriptions. 

```suggestion
`provisioningNetwork` | `Managed` | ProvisioningNetwork is used to indicate if there is a network dedicated to provisioning traffic, and how it will be managed. Valid values are `Managed` (default), `Unmanaged`, and `Disabled`. See <link to new section describing the options in more detail> for more detail.
```